modules/tinymce/src/core/main/ts/delete/BlockRangeDelete.ts

```diff
@@ -87,7 +87,7 @@ const deleteSymbol = (editor: Editor): Optional<() => void> => {
     return Optional.none();
   }
   const editableDiv = getClosestEditableDiv(editor);
-  if(editableDiv?.textContent?.length == 1){
+  if(editableDiv?.textContent?.length == 1 || editableDiv?.textContent?.length == 0){
```

Comment:
Funguje to hezky, akorát zjednodušit podmínku:
```suggestion
  if(editableDiv?.textContent?.length <= 1){
```